### PR TITLE
indent nested menu according to the existance of icons in items, and …

### PR DESCRIPTION
**Pull request by hakifran**
…also highlight parent when there is a child path which has parameters in it
refactor the code which check if at least one item has icon

### CODE REVIEW

File: src/components/NestedAccordionMenu/MenuItems/index.js [hunk 1]
```diff
@@ -32,10 +33,18 @@ function MenuItem(props) {
...
 
+  function hasIcons(subItems) {
+    subItems.forEach(itm => {
+      if (itm.Icon) {
+        hasIcon = true;
+      }
+    });
+    return hasIcon;
+  }
+
```
Comment (borisrorsvort) [inline src/components/NestedAccordionMenu/MenuItems/index.js]: I’d extract this to a new file in https://github.com/BLSQ/manager-ui/tree/master/src/utils as you’re also using that in https://github.com/BLSQ/manager-ui/pull/9/files#diff-03c0bafcbd11ad5ac0301f2b346cb7edR16

**Review by borisrorsvort — CHANGES_REQUESTED** [review]

File: src/components/NestedAccordionMenu/MenuItems/index.js (hunk 1 above)
Comment (vanakenm) [reply src/components/NestedAccordionMenu/MenuItems/index.js]: If this is about knowing if at least one subItems has an icon (and updating hasIcon) accordingly maybe use some: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some ?

hasIcon = subItems.some((itm) => itm.Icon)
return hasIcon

Comment (vanakenm) [reply src/components/NestedAccordionMenu/MenuItems/index.js]: (and at this time, evaluate if a function is still useful as it's a one liner)

Comment (hakifran) [reply src/components/NestedAccordionMenu/MenuItems/index.js]: @vanakenm thank you, I have changed and now there is no need to use a function anymore

**Review by borisrorsvort — APPROVED** [review]

**Pull request merged**